language_names/manual.py

```diff
@@ -0,0 +1,32 @@
+import sqlite3
+import textwrap
+import csv
+
+
+def populate_database():
+    conn = sqlite3.connect('test.db')
+    c = conn.cursor()
+    c.execute(textwrap.dedent("""
+        PRAGMA foreign_keys=OFF;"""))
+    c.execute(textwrap.dedent("""
+        BEGIN TRANSACTION;"""))
+    c.execute(textwrap.dedent("""
+        CREATE TABLE IF NOT EXISTS fixes (id integer primary key, lg text, inLg text, name text, unique(lg, inLg) on conflict replace);
+        """))
+    with open('fixes.tsv', 'r') as f:
```

Comment:
This loop would probably be best extracted into a function since you do the same thing twice here and you'll have to do it again (for the large SQL file).

Reply:
Okay I have written a function for it.